modules/core/shared/src/main/scala/gem/enum/MagnitudeBand.scala

```diff
@@ -0,0 +1,67 @@
+  /** @group Constructors */ case object U extends MagnitudeBand("U", "U", "Ultraviolet", gem.math.Wavelength.unsafeFromAngstroms(3600), gem.math.Wavelength.unsafeFromAngstroms(750), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object B extends MagnitudeBand("B", "B", "Blue", gem.math.Wavelength.unsafeFromAngstroms(4400), gem.math.Wavelength.unsafeFromAngstroms(900), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object V extends MagnitudeBand("V", "V", "Visual", gem.math.Wavelength.unsafeFromAngstroms(5500), gem.math.Wavelength.unsafeFromAngstroms(850), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object Uc extends MagnitudeBand("Uc", "UC", "UCAC", gem.math.Wavelength.unsafeFromAngstroms(6100), gem.math.Wavelength.unsafeFromAngstroms(630), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object R extends MagnitudeBand("R", "R", "Red", gem.math.Wavelength.unsafeFromAngstroms(6700), gem.math.Wavelength.unsafeFromAngstroms(1000), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object I extends MagnitudeBand("I", "I", "Infrared", gem.math.Wavelength.unsafeFromAngstroms(8700), gem.math.Wavelength.unsafeFromAngstroms(1000), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object Y extends MagnitudeBand("Y", "Y", "Y", gem.math.Wavelength.unsafeFromAngstroms(10200), gem.math.Wavelength.unsafeFromAngstroms(1200), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object J extends MagnitudeBand("J", "J", "J", gem.math.Wavelength.unsafeFromAngstroms(12500), gem.math.Wavelength.unsafeFromAngstroms(2400), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object H extends MagnitudeBand("H", "H", "H", gem.math.Wavelength.unsafeFromAngstroms(16500), gem.math.Wavelength.unsafeFromAngstroms(3000), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object K extends MagnitudeBand("K", "K", "K", gem.math.Wavelength.unsafeFromAngstroms(22000), gem.math.Wavelength.unsafeFromAngstroms(4100), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object L extends MagnitudeBand("L", "L", "L", gem.math.Wavelength.unsafeFromAngstroms(37600), gem.math.Wavelength.unsafeFromAngstroms(7000), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object M extends MagnitudeBand("M", "M", "M", gem.math.Wavelength.unsafeFromAngstroms(47700), gem.math.Wavelength.unsafeFromAngstroms(2400), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
+  /** @group Constructors */ case object N extends MagnitudeBand("N", "N", "N", gem.math.Wavelength.unsafeFromAngstroms(104700), gem.math.Wavelength.unsafeFromAngstroms(52300), gem.enum.MagnitudeSystem.unsafeFromTag("Vega"))
```

Comment:
Can you use references for enum values; i.e., instead of `gem.enum.MagnitudeSystem.unsafeFromTag("Vega")` generate `gem.enum.MagnitudeSystem.Vega`?

Reply:
I was trying something similar but didn't manage to get it working. I'll fiddle a bit more.